dashboard/src/app/[tenantId]/components/LinkWithTenant.tsx

```diff
@@ -1,11 +1,18 @@
 'use client'
+import { cn } from '@/components/utils'
 import NextLink from 'next/link'
 import { useParams } from 'next/navigation'
 import { ComponentProps } from 'react'
 
-const LinkWithTenant = ({ ...props }: ComponentProps<typeof NextLink>) => {
+const LinkWithTenant = ({ linkStyle, ...props }: ComponentProps<typeof NextLink> & { linkStyle?: boolean }) => {
   const { tenantId } = useParams()
-  return <NextLink {...props} href={`/${tenantId}${props.href}`} />
+  return (
+    <NextLink
+      {...props}
+      href={`/${tenantId}${props.href}`}
+      className={cn(props.className, { 'text-blue-500 underline': linkStyle })}
```

Comment:
why not just pass "text-blue-500 underline" when using the component. I would be okay with this if we were using this in many places but right now its just one? Even if we were using in many places I don't see a problem with just referencing it there instead of the main component.

Reply:
even though the hyperlink style isn't being used in multiple places, it is still a likely styling choice that will be used in the future. and adding a prop to style it was intended to avoid the possibility of links being styled differently across the app. like, what if someone used `text-blue-300` instead of `text-blue-500`, or what if they didn't add an underline. 

minor stuff, but just remaining consistent was the idea.

however, upon further review with your other comments, I believe using the `shadcn's` `link` `variant` provides an even greater global styling enforcement across the app.

Reply:
thats a good point i forgot about the the `link` `variant`. I agree with your first paragraph if there was not variant though.

Reply:
a majority of the `LinkWithTenant` references include `className's` with the same exact styling across all these different files and use cases. I think resolving this issue by implementing `shadcn's` `link` `variant` `Button` is due for a refactor in its own separate issue & PR.

for now, I have implemented a component that fits for a smaller use case within `UserTask.tsx`. 

<img width="281" alt="image" src="https://github.com/user-attachments/assets/8ddb2fc5-6c42-4cf0-af8e-d089a0b92b3e">